<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSearchs</class>
 <widget class="QDialog" name="CustomSearchs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="searchEngineItems">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addItemButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteItemButton">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>440</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="turnUpButton">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>440</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="turnDownButton">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>440</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlLbale">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>url</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlLineEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="firstHotKey">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="secondHotKey">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="thirdHotKey">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enableButton">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>440</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>enable</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomSearchs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomSearchs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
